frontend/src/pages/team/Settings/General.vue

```diff
@@ -140,6 +147,25 @@ export default {
 
         async fetchData () {
             this.cancelEditName()
+        },
+        checkSlug () {
+            if (this.pendingSlugCheck) {
+                clearTimeout(this.pendingSlugCheck)
+            }
+            this.pendingSlugCheck = setTimeout(() => {
```

Comment:
[nb] There's now a debounce like this at least a couple of places now (Users/Genera, Users/Team), it might be worth extracting it into its own function, particularly as then it removes the extra attr in the Vue state.

Reply:
Raised as a future task